src/components/MenuProductModal/MenuProductModal.tsx

```diff
@@ -0,0 +1,125 @@
+                  <p className="font-poly text-left italic text-[#333F4C]">{product.description}</p>
+                  <div className="flex w-full items-center justify-between gap-2">
+                    <div className="font-semibold text-[#333F4C]">V, Ve, GF</div>
+                    <div className="flex flex-row items-center justify-between gap-2 text-[#333F4C]">
+                      <button className="flex h-10 w-10 items-center gap-3.5 rounded-md bg-[#E4EAEB] p-4" onClick={decrease}>
+                        -
+                      </button>
+                      <div>{counter}</div>
+                      <button className="flex h-10 w-10 items-center gap-3.5 rounded-md bg-[#E4EAEB] p-4" onClick={increase}>
+                        +
+                      </button>
+                    </div>
+                  </div>
+                </div>
+              </div>
+
+              <div className="flex w-full flex-col justify-items-start gap-2 p-0">
+                <div className="font-poly text-left text-base font-normal italic text-[#A2AAB6]">Contains Allergens</div>
+                <div className="flex flex-row justify-items-start">
+                  {allergens.map((allergen) => (
+                    <>
+                      <label key={allergen} className="gap-2 p-2 text-[#333F4C]">
+                        {allergen}
+                      </label>
+                    </>
+                  ))}
+                </div>
+              </div>
+
+              <div className="flex flex-row items-center gap-4 self-stretch rounded-lg p-0">
+                <Button
+                  className="gap-2.5py-4 flex grow justify-center border-2 border-solid border-[#A2AAB6] px-3"
+                  size="small"
+                  colour="clear"
+                  onClick={() => disableShow(false)}
+                >
+                  Cancel
+                </Button>
+                {counter > 0 ? (
+                  <Button
+                    onClick={() => {
+                      addToOrder();
+                      disableShow(false);
+                    }}
+                    size="small"
+                    colour="yellow"
+                    className="gap-2.5py-4 flex grow justify-center px-3"
+                  >
+                    Add to order £{(product.price * counter * 1e2) / 1e2}
+                  </Button>
+                ) : (
+                  <Button
+                    onClick={() => {
+                      removeProduct(product);
+                      disableShow(false);
+                    }}
+                    size="small"
+                    colour="yellow"
+                    className="gap-2.5py-4 flex grow justify-center px-3"
+                  >
+                    Remove from order
+                  </Button>
+                )}
+              </div>
+            </div>
+          </div>
+        </>
+      ) : null}
```

Comment:
should have a message here, saying something like 'No allergens' - maybe a bit more elaborate than that

Reply:
this still needs adding

Reply:
Oh yes, in that case just have - showmodal && ()